### PR TITLE
e2e-484 get team office locations

### PR DESCRIPTION
**Pull request by axle-h**
image

### CODE REVIEW

**Comment by axle-h** [comment]
Also aded spring profile for using a local file H2 database, this helps massively with application start-up & for using intellij database tools to connect.

**Review by electrified — APPROVED** [review]
LGTM

File: src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/TeamAPITest.java [hunk 1]
```diff
@@ -95,6 +99,93 @@ public void canGetNextUpdateWithDateChanged() {
...
+            .auth().oauth2(token)
+            .when()
+            .get("/teams/C04000/office-locations")
+            .then()
+            .assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("size()", greaterThan(0))
+            .root("find { it.code == '%s' }")
+
+            .body("description", lincoln, Matchers.equalTo("Lincoln office"))
```
Comment (andymarke) [inline src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/TeamAPITest.java:119]: I love this "withArgs" thing :-)

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**